jib-core/src/main/java/com/google/cloud/tools/jib/docker/json/DockerLoadManifestTemplate.java

```diff
@@ -39,14 +39,18 @@
  * @see <a href="https://github.com/moby/moby/blob/master/image/tarexport/load.go">Docker load
  *     source</a>
  */
-class DockerLoadManifestTemplate implements JsonTemplate {
+public class DockerLoadManifestTemplate implements JsonTemplate {
 
   private final String config = "config.json";
-  private final List<String> repoTags = new ArrayList<>(1);
+  private final List<String> repoTags = new ArrayList<>();
```

Comment:
Initial capacity can still be 1 since otherwise it allocates 10 for initial capacity be default.